docs/bugzilla.md

```diff
@@ -29,7 +29,8 @@ fields.
 
 ```python
 t.create(summary='Ticket summary',
-         description='Ticket description')
+         description='Ticket description',
+         groups='Group Name')
```

Comment:
Remove this line: groups='Group Name' 
But add groups under Tested create() ticket fields